### DISCUSSION TITLE
Property is ignored due to the display. With 'display: block', vertical-align should not be used.

### DISCUSSION THREAD

**Discussion by Jo75zx** [tailwindlabs/tailwindcss#14297] [Help]
![image](https://github.com/user-attachments/assets/01df31a6-08c0-4265-8cc3-0b9e5e59948f)

![image](https://github.com/user-attachments/assets/2cc09a95-26f0-4e9e-8e1a-f219eeeabfa1)

![image](https://github.com/user-attachments/assets/b6d4dffd-e650-480e-bd98-2fa69446aad6)

![image](https://github.com/user-attachments/assets/34137d06-ba33-448c-8cc4-f6f00b5add23)

### Hey I just installed tailwind and i got this error in my editor ,but isn't this the official code , thanks in advance

**Comment by wongjn** [comment]
Those aren't errors per-se, rather warnings . I believe each of the declarations serve a purpose. [See the annotated source file](https://github.com/tailwindlabs/tailwindcss/blob/v3.4.10/src/css/preflight.css) for reasoning for each declaration.
